Specification/4._CAN_bus_transport_layer.md

```diff
@@ -19,43 +19,39 @@ UAVCAN defines the following types of transfers:
 Both message and service transfers can be further distinguished between:
 
 * Single-frame transfer - a transfer that is entirely contained in a single CAN frame.
-* Multi-frame transfer - a transfer whose payload is distributed over multiple CAN frames.
+* Multi-frame transfer - a transfer that has its payload distributed over multiple CAN frames.
 
 The following properties are common to all types of transfer:
 
 Property        | Description
 ----------------|------------------------------------------------------------------------------------------------------
 Payload         | The serialized data structure
-Data type ID    | This identifier indicates how the data structure should be interpreted
-Priority        | See below
-Transfer ID     | An integer value that allows receiving nodes to distinguish this transfer from others
+Data type ID    | An identifier that indicates how the data structure should be interpreted
+Priority        | An positive integer value that defines the message urgency (higher numbers indicate reduced priority). Higher priority transfers can delay transmission of loser priority transfers. 
+Transfer ID     | An integer value that allows receiving nodes to distinguish this transfer from all others
 
-The transfer priority is an integer number that defines the urgency of the data contained in the transfer.
-Numerically lower priority values indicate higher urgency, and numerically higher values indicate lower urgency.
-Transfers of higher priority (i.e. with numerically lower priority value) can delay transmission of transfers
-whose priority are lower (i.e. numerically higher).
 
 ### Message broadcasting
 
-A broadcasted message is carried by a single message transfer that contains the serialized message.
-A broadcasted message has the following properties:
+A broadcast message is carried by a single-frame transfer that contains the serialized message.
+A broadcast message has the following properties:
 
 Property        | Description
 ----------------|------------------------------------------------------------------------------------------------------
 Payload         | The serialized message
-Data type ID    | This identifier indicates how the message should be interpreted
+Data type ID    | An identifier that indicates how the message should be interpreted
 Source node ID  | Node ID of the node that has transmitted the transfer
 Priority        | See above
 Transfer ID     | See above
 
 In order to broadcast a message, the broadcasting node must have a node ID that is unique within the network.
-An exception is applied to *anonymous message transfers*.
+An exception is applied to *anonymous message broadcasts*.
```

Comment:
FYI, changed this to better align it with the heading that follows.

Reply:
This is an interesting case, because in this context "transfer" and "message broadcast" mean the same thing, so it is fine to change it like that.